### ISSUE TITLE
On Linux, character camera sometimes cannot turn around fully if in fullscreen

### ISSUE THREAD

**Issue by yaomtc** [LWJGL/lwjgl#115]
As described in this Minecraft bug report:
https://bugs.mojang.com/browse/MC-80282
and on this reddit thread:
http://www.reddit.com/r/linux_gaming/comments/35nheg
sometimes in fullscreen the camera (and thus the character) cannot turn all the way around, but only as far as the invisible cursor can go to the edge of the screen. This doesn't happen all the time, and this behavior goes away when you view the Esc menu, or leave fullscreen and return to it.

**Comment by tfg13** [comment]
Bounced this back to Mojang (see explanation in minecraft tracker)

tl;dr: We don't know what Minecraft exactly calls in LWJGL, and without the code it would require an awful amount of time & effort to figure it out. Someone at Mojang will have to look into what happens when ESC is pressed and if it really turns out to be an LWJGL issue, I will look into this again.

**Comment by lehjr** [comment]
I ran into this issue with Minecraft 1.11.2 today, their latest release. Sad to see it unresolved 2 years later.

**Comment by philipwhiuk** [comment]
I have to say I'm very disappointed in Mojang's behaviour on that bug. They are responsible for upstreaming issues with a SSCCE showing the LWJGL behaviour. It's not up to LWJGL developers to divine the calls they are making in their code based on a bug reported to them about how Minecraft functions.

**Comment by lehjr** [comment]
I have to agree. They totally dropped the ball. Also, they are shipping a nightly build with 1.11.2, but not the latest nightly build. Edit: their version: lwjgl-2.9.4-nightly-20150209.jar, but the latest would have been Build #176 (Jul 28, 2015 9:37:09 AM)

**Comment by lehjr** [comment]
Just tested this under windows with Minecraft and the issue is still there, but slightly different. In windows 10, the player can rotate a few degrees like in Linux and stops, but then unlike Linux, the entire view rotates and then the player can rotate. It's almost like "lag" but the rotation was consistent where it would stop, not to mention debugging output was showing a steady 60fps. Leaving full screen returns the rotation functionality to normal but the window can no longer be resized. I doubt any of that information can help isolate the problem, but it does show that it is more than one OS.
